### PR TITLE
[explorer] Now wait for just one API call to display

### PR DESCRIPTION
**Pull request by Andrew47**
Reduces Load Time in the Home Page by using getRecentTransactions API function. 

Before:

https://user-images.githubusercontent.com/11377188/179023268-8e83852e-1ecf-4dcb-b398-658f5a00fc44.mp4

After:

https://user-images.githubusercontent.com/11377188/179023497-17ccc084-cdb4-4570-884c-1ff21796d93a.mp4

### CODE REVIEW

**Review by Jibz1 — APPROVED** [review]

**Review by 666lcz — CHANGES_REQUESTED** [review]
Thanks for working on this! 

I am a bit concerned about special casing on transaction count == 0 because this assumption will break on a fresh/local network

File: explorer/client/src/components/transaction-card/RecentTxCard.tsx [hunk 1]
```diff
@@ -77,6 +77,13 @@ async function getRecentTransactions(
     pageNum?: number
 ): Promise<TxnData[]> {
     try {
+        if (totalTx === 0) {
```
Comment (666lcz) [inline explorer/client/src/components/transaction-card/RecentTxCard.tsx:80]: Abusing  the meaning of`totalTx` here is a bit confusing to the reader, I would move line 81 - 85 to line 262 with the condition that `pagedNum === 1`

File: explorer/client/src/utils/numberUtil.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 // Number Suffix
 export const numberSuffix = (num: number): string => {
+    // Handle case where count not found:
+    if (num === 0) return '-';
```
Comment (666lcz) [inline explorer/client/src/utils/numberUtil.ts:7]: We should not do this in a general purpose util function. What if the caller just want to display "0 transactions"?

Comment (Andrew47) [reply explorer/client/src/utils/numberUtil.ts]: Done

File: explorer/client/src/components/transaction-card/RecentTxCard.tsx (hunk 1 above)
Comment (Andrew47) [reply explorer/client/src/components/transaction-card/RecentTxCard.tsx]: Done

**Comment by Andrew47** [comment]
@666lcz --> this PR is ready for another review. Following feedback, I've removed dependencies on unusual counts / transaction numbers. This means a different flow to how the Home Page works that is explained in added code comments.

**Comment by 666lcz** [comment]
@Andrew47 , thanks for addressing the comments and sorry for the delay in review. It looks like we are duplicating a lot of code in this PR and I am wondering if there's better way to avoid that. Will take a closer look today

**Review by randall-Mysten — APPROVED** [review]

**Pull request closed without merging**